orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -190,6 +207,13 @@ def __init__(self, module):
         self._save_onnx = False
         self._save_onnx_prefix = ''
 
+        # CPP extension to get torch CUDA allocator's alloc and free function addresses
+        self._use_external_cuda_allocator = True
+        if self._use_external_cuda_allocator:
```

Comment:
Is it allowed to do something like

```
model1 = ORTModule(model1)
model2 = ORTModule(model2)
```
in the same process (python interpreter)? Just checking `load_inline` or self._torch_cuda_allocator.cuda_caching_allocator_raw_delete_address()` can handle this

Reply:
I haven't personally tried but I don't see why it wouldn't work. It will just recompile and recreate the binary file.